src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -48,6 +58,12 @@ def get_crash_data(crash_data,
   unless |symbolize| flag is set to False. Symbolized stacktrace will contain
   inline frames, but we do exclude them for purposes of crash state generation
   (helps in testcase deduplication)."""
+
+  # For Android LKL (and potentially kernel output), the KASAN crash may start
+  # with the time since boot.  We need to remove this so that our regexes match.
+  if '] BUG: KASAN:' in crash_data:
```

Comment:
Add a check to make sure this is Android.

Reply:
+1

Reply:
LKL runs on host so its not Android platform sadly.  I am asking xuanxing@ if they could add "LKL" or something to the crash stack so we can identify it.

Reply:
The job type is selected to use android platform right, so environment.is_android() should still work.

Reply:
No its a host based fuzzer, does not run on an android phone runs on x86 GCE host

Reply:
We have added 'Linux Kernel Library Stack Trace:' so we know explicitly now